Solutions/MSTelemetrySDK.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.28307.645
+# Visual Studio Version 16

```

Comment:
I usually don't commit that change because I am not 100% sure how that'd be handled by Visual Studio 2017.

Reply:
Ah, that's what is generating it. Eh, I can revert this.